### ISSUE TITLE
"Complete" mature example including telemetry (logging, metrics, tracing)

### ISSUE THREAD

**Issue by Joannis** [hummingbird-project/hummingbird-examples#117]

**Comment by adam-fowler** [comment]
What do you mean by mature? We can have an example that enables all of these and then provide details on how to setup services to view the results.

**Comment by Joannis** [comment]
Exactly, I wrote this quickly at the conference, but we were discussing that current examples show integrating one library but not all the ones you would want. So we should have one or more "highlighted" production example project(s) that also include observability and any other things every mature app should have. Then of course we would want docs on setup as you said, plus some  deployment scripts and docs.

**Comment by adam-fowler** [comment]
Well I guess todos-auth-fluent would be the starting point.

**Comment by adam-fowler** [comment]
I wanted to do a full authentication example at some point ie
- user/password
- forgot password flow
- add OTP
- add pass key
- oauth (hopefully Tibor will release his oauth flow sometime soon)

**Comment by adam-fowler** [comment]
#137
